IBAnimatable.playground/Pages/Chaining Animations.xcplaygroundpage/Contents.swift

```diff
@@ -15,13 +15,27 @@ let view = CircleView()
 iPhoneView.addSubview(view)
 
 ////: Start another animation in completion closure
-view.squeezeInDown { view.pop { view.shake { view.squeeze { view.wobble { view.flipX { view.flash { view.flipY { view.fadeOutDown() } } } } } } } }
+view.squeezeFade(.in, direction: .down) {
+    view.pop(repeatCount: 1) {
+        view.shake(repeatCount: 1) {
+            view.squeeze(repeatCount: 1) {
+                view.wobble(repeatCount: 1) {
+                    view.flip(axis: .x) {
+                        view.flip(axis: .y) {
+                            view.slideFade(.out, direction: .down)
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
It looks horrible now, off cause not your fault, I think the next big thing can be trainable animation methods.